CefSharp.Example/RequestHandler.cs

```diff
@@ -137,8 +137,15 @@ bool IRequestHandler.OnSelectClientCertificate(IWebBrowser browserControl, IBrow
             // We also suggest you explicitly Dispose of the callback as it wraps an unmanaged resource.
 
             callback.Dispose();
-            //X509Certificate2Collection selectedCertificateCollection = System.Security.Cryptography.X509Certificates.X509Certificate2UI
             return false;
+
+            // This is an alternate for displaying a Client Certificate Dialog for user to select a certificate
+            //X509Certificate2Collection selectedCertificateCollection = X509Certificate2UI.SelectFromCollection(certificates, "Certificates Dialog", "Select Certificate for authentication", X509SelectionFlag.SingleSelection);
```

Comment:
Yes, this works. This should provide a default dialog but not sure if that will work for WPF?